src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml

```diff
@@ -67,14 +68,14 @@ tests:
       command: "writeAttribute"
       attribute: "on level"
       arguments:
-          value: 254
+          value: 5
 
     - label: "Reads the OnLevel attribute from the DUT"
       PICS: LVL.S.A0011
       command: "readAttribute"
       attribute: "on level"
       response:
-          value: 254
+          value: 5
           constraints:
               type: uint8
               notValue: OnLevelValue
```

Comment:
What if the original value was 5?

Reply:
@bzbarsky-apple i will update the script with configurable values